src/js/features/emailNotifications/saga.js

```diff
@@ -0,0 +1,89 @@
+/* global web3 */
+
+import {call, fork, put, takeEvery} from 'redux-saga/effects';
+import {
+  CHECK_EMAIL_SUBSCRIPTION, CHECK_EMAIL_SUBSCRIPTION_SUCCESS, CHECK_EMAIL_SUBSCRIPTION_FAILURE,
+  SUBSCRIBE_EMAIL, SUBSCRIBE_EMAIL_FAILURE, SUBSCRIBE_EMAIL_SUCCESS,
+  VERIFY_EMAIL, VERIFY_EMAIL_FAILURE, VERIFY_EMAIL_SUCCESS,
+  UNSUBSCRIBE_EMAIL, UNSUBSCRIBE_EMAIL_FAILURE, UNSUBSCRIBE_EMAIL_SUCCESS
+} from './constants';
+import axios from 'axios';
+
+const DAPP_ID = 'teller-network';
```

Comment:
What do you think of moving the constants related to the service URL to a constants file inside `src/constants`?